core/Cargo.toml

```diff
@@ -9,6 +9,7 @@ repository = "https://github.com/solana-labs/solana"
 authors = ["Solana Maintainers <maintainers@solana.com>"]
 license = "Apache-2.0"
 edition = "2018"
+build = "../sdk/build.rs"
```

Comment:
I suspect this'll break once the core crate is published to crates.io, as `../sdk/` won't exist

Reply:
I think we sadly need to duplicate `sdk/build.rs` to all consumers.  I wonder if we can use a symlink, and if so what `cargo publish` will do with that symlink.  (ie, core/build.rs is a symlink to ../sdk/build.rs)

Reply:
@mvines Luckily it seems that we can expect cargo to resolve symlinks to regular files, in general. :)
Comment from the rust-lang member: https://github.com/rust-lang/cargo/issues/3896#issuecomment-457744503
Live and prominet example: https://github.com/serde-rs/serde/tree/bb99b31eb0a55393101f9c80cd959b3a739ad70f/serde